modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryFolderLocalServiceImpl.java

```diff
@@ -250,6 +258,36 @@ private Map<Locale, String> _getLabelMap(
 		return labelMap;
 	}
 
+	private void _updateWorkflowDefinitionLinks(
+			long objectEntryFolderId, ServiceContext serviceContext)
+		throws PortalException {
+
+		if (!GetterUtil.getBoolean(
+				serviceContext.getAttribute("updateWorkflowDefinitionLinks"),
+				true)) {
+
+			return;
+		}
+
+		List<ObjectValuePair<Long, String>> workflowDefinitionOVPs =
+			new ArrayList<>();
+
+		String workflowDefinition = ParamUtil.getString(
+			serviceContext,
+			"workflowDefinition" +
+				ObjectDefinitionConstants.OBJECT_DEFINITION_ID_ALL);
+
+		workflowDefinitionOVPs.add(
+			new ObjectValuePair<Long, String>(
+				ObjectDefinitionConstants.OBJECT_DEFINITION_ID_ALL,
+				workflowDefinition));
```

Comment:
Can you use `Collections.singletonList` here and inline as much as SF allows?

Reply:
done, and rebased

Thanks!